### ISSUE TITLE
Erreur de protocole XML for Analysis : l'élément « schema » dans l'espace de noms « http://www.w3.org/2001/XMLSchema » est attendu.

### ISSUE THREAD

**Issue by Dialloahm** [DaxStudio/DaxStudio#1312]
DAX Studio v3.2.1.1046
Connection: PowerPivot - 13.0.1700.1121

## Steps to Reproduce
1.
2.
24/09/2024 19:46:40		Error Exporting 'Bilan':  Erreur de protocole XML for Analysis : l'élément « schema » dans l'espace de noms « http://www.w3.org/2001/XMLSchema » est attendu.

**Comment by dgosbell** [comment]
Is this coming from the "Export Data" function? Does it happen all the time or just occasionally? I've just done a quick test to double check this and it's not happening on my machine when exporting from PowerPivot.

**Comment by Dialloahm** [comment]
Yes this coming from the "Export Data" function. It's just happen when i try to export a table whit 65 millions rows from power pivot. I don't have this problem when the number of rows is 5 millions. How can I do to export 65 millions rows from power pivot in one  cvs file?

**Comment by dgosbell** [comment]
I don't have a data model quite that large, but I've tested with a model that has 21 million rows and that worked without error.
